packages/fuels-core/src/lib.rs

```diff
@@ -66,6 +67,7 @@ impl ParamType {
             Self::Array(_params, _l) => true,
             // The other primitive types are inside `Return`,
             // thus smaller than one `WORD`.
+            Self::Tuple(_) => true,
```

Comment:
I think we should check if `len(Tuple) > 1`

Reply:
Sure, by the way, this whole block is going away [very soon](https://github.com/FuelLabs/fuels-rs/issues/201).